src/View/pagination.html.twig

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+  <title>Message Board</title>
+</head>
+<body>
+
+{% if lastPage > 1 %}
+
+    {# the number of first and last pages to be displayed #}
+    {% set extremePagesLimit = 3 %}
+
+    {# the number of pages that are displayed around the active page #}
+    {% set nearbyPagesLimit = 2 %}
+
+    <div class="pagination pagination-centered">
+        <ul>
+          {% if currentPage > 1 %}
+            <li><a href="{{ paginationPath }}{{ currentPage-1 }}">&larr;</a></li>
+
+            {% for i in range(1, extremePagesLimit) if ( i < currentPage - nearbyPagesLimit ) %}
+                <li><a href="{{ paginationPath }}{{ i }}">{{ i }}</a></li>
+            {% endfor %}
+
+            {% if extremePagesLimit + 1 < currentPage - nearbyPagesLimit %}
+                <li class="disabled"><a href="#">...</a></li>
+            {% endif %}
+
+            {% for i in range(currentPage-nearbyPagesLimit, currentPage-1) if ( i > 0 ) %}
```

Comment:
IMO views should really not be doing any calculations. All they should be doing is querying what data is available and then displaying it accordingly.

Reply:
On second thoughts, this looks like it would be good as a Twig extension. If we add a Todo/Task here to make this into an Extension (or better yet use one that is already available) then I'd be happy with this as a temporary solution

Reply:
I don't know how to do that... @warslett ?

Reply:
added a new issue to look at it another time: https://github.com/BiffBangPow/message-board/issues/11